plugins/ui-kit-candidates/HeaderBar.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { coreColors } from "@dcos/ui-kit/dist/packages/shared/styles/color";
+
+const headerBarStyles: object = {
+  backgroundColor: coreColors().purpleDarken4,
+  height: "32px",
+  width: "100%",
+  flexShrink: 1,
+  flexGrow: 0,
+  flexBasis: "auto",
+  zIndex: 999
+};
+
+export class HeaderBar extends React.PureComponent {
+  render() {
+    const { children } = this.props;
+    return <div style={headerBarStyles}>{children}</div>;
```

Comment:
🎨 this is one of the rare cases where I like `this.props.children` better 😆

Reply:
agree 😆 there was more code before which I managed to strip out - that last refactoring slipped. I'll change it if I touch the file again.